content/applications/sales/sales/products_prices/prices/pricing.rst

```diff
@@ -317,6 +317,38 @@ product form.
 
 .. _sales/advanced-price-rules:
 
+Discounts
+=========
+
+The :guilabel:`Discounts` feature allows the ability to set a discount or increase the price
+on *individual items* in a sales order. This is calculated as a percentage of that product's sales
+price.
+
+To access discounts, navigate to the *Sales* app setting page
+(:menuselection:`Sales app --> Configuration --> Settings`), and click the :guilabel:`Discounts`
+checkbox, then click :guilabel:`Save`.
+
+.. image:: pricing/discounts-option.png
+   :align: center
+   :alt: Click the checkbox labeled `Discounts` to activate Discounts.
+
+To set a discount after discounts have been activated, navigate to the list of sales orders by
+going to :menuselection:`Orders --> Orders` at the top of the page. Once there, click on the
```

Comment:
Good practice to add in the app name at the beginning, even if we're in the app already.
This is because documentation readers tend to skim and also read content out of order, so if they land  here, they get the full address without having to look for context clues.

```suggestion
going to :menuselection:`Sales app --> Orders --> Orders` at the top of the page. Once there, click on the
```

...or if you consider the previous feedback around quotations (vs. sales orders)
```suggestion
going to :menuselection:`Sales app --> Orders --> Quotations` at the top of the page. Once there, click on the
```